tests/reg_issue960_npe_resolve_syntactic_sugar2/test_issue960.fz.expected_err

```diff
@@ -0,0 +1,16 @@
+
+--CURDIR--/test_issue960.fz:8:5: error 1: Failed to infer result type for feature 'test_ambig.f'.
+    f := ()->x
+----^
+To solve this, please specify a result type explicitly.
+
+
+--CURDIR--/test_issue960.fz:8:12: error 2: No type information can be inferred from a lambda expression
+    f := ()->x
+-----------^
+A lambda expression can only be used if assigned to a field or argument of type 'Function'
+with argument count of the lambda expression equal to the number of type parameters of the type.  The type of the
+assigned field must be given explicitly.
+To solve this, declare an explicit type for the target field, e.g., 'f (i32, i32) -> bool := x, y -> x > y'.
+
+2 errors.
```

Comment:
This could be another enhancement request, or ma<be even two

- first, this example should produce only one error, not two
- second, it should actually be possible to infer the type of a nullary lambda expression as in this case, problematic are only the arguments.
Ok, for now, but can you add issues for these and maybe refer to them in the test?